### PR TITLE
lmb-1028: create migration to set gemeenteraadslid to lid in rangorde

### PR DESCRIPTION
**Pull request by JonasVanHoof**
## Description

We change the rangorde label of GR to be lid instead of gemeenteraadslid. Create a migration to update all

## How to test

- In IV create some mandatarissen to have a rangorde _gemeenteraadslid_ of _Gemeenteraadslid_ 
- Let the migrations run
- See if they are updated (restart cache and resource)

### CODE REVIEW

File: config/migrations/2024/20241108111400-change-rangorde-TS-to-schepen.sparql [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+    VALUES ?schepenMandaat {
+      bestuursfunctieCode:5ab0e9b8a3b2ca7c5e000014 # MANDAAT_SCHEPEN_CODE
+      bestuursfunctieCode:59a90e03-4f22-4bb9-8c91-132618db4b38 # MANDAAT_TOEGEVOEGDE_SCHEPEN_CODE
+      bestuursfunctieCode:5ab0e9b8a3b2ca7c5e00001e # MANDAAT_DISTRICT_SCHEPEN_CODE
+      bestuursfunctieCode:5ab0e9b8a3b2ca7c5e000020 # MANDAAT_GEDEPUTEERDE_CODE
+    } 
+    ?mandaat org:role ?schepenMandaat.
+
+    FILTER regex(str(?rangorde), "(Toegevoegde schepen|Districtsschepen|Gedeputeerde)$")
```
Comment (JonasVanHoof) [inline config/migrations/2024/20241108111400-change-rangorde-TS-to-schepen.sparql:30]: Got these labels from the bestuursfunctie codes. Could not find an example of a user filling it in different than these values

File: config/migrations/2024/20241108111400-change-rangorde-TS-to-schepen.sparql [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    ?mandataris a mandaat:Mandataris.
+    ?mandataris org:holds ?mandaat.
+    ?mandataris mandaat:rangorde ?rangorde.
+
+
+    VALUES ?schepenMandaat {
+      bestuursfunctieCode:5ab0e9b8a3b2ca7c5e000014 # MANDAAT_SCHEPEN_CODE
+      bestuursfunctieCode:59a90e03-4f22-4bb9-8c91-132618db4b38 # MANDAAT_TOEGEVOEGDE_SCHEPEN_CODE
+      bestuursfunctieCode:5ab0e9b8a3b2ca7c5e00001e # MANDAAT_DISTRICT_SCHEPEN_CODE
+      bestuursfunctieCode:5ab0e9b8a3b2ca7c5e000020 # MANDAAT_GEDEPUTEERDE_CODE
```
Comment (JonasVanHoof) [inline config/migrations/2024/20241108111400-change-rangorde-TS-to-schepen.sparql:26]: https://github.com/lblod/frontend-lokaal-mandatenbeheer/blob/master/app/models/mandaat.js#L62

File: config/migrations/2024/20241108111400-change-rangorde-TS-to-schepen.sparql (hunk 1 above)
Comment (Rahien) [inline config/migrations/2024/20241108111400-change-rangorde-TS-to-schepen.sparql:30]: let's make it consistent and also replace Schepen (capital) with schepen?

File: config/migrations/2024/20241108111400-change-rangorde-TS-to-schepen.sparql (hunk 2 above)
Comment (Rahien) [reply config/migrations/2024/20241108111400-change-rangorde-TS-to-schepen.sparql:26]: let's remove gedeputeerde here. I will create a ticket to fix the rank for them too

File: config/migrations/2024/20241108111400-change-rangorde-TS-to-schepen.sparql [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  GRAPH ?graph {
+  ?mandataris mandaat:rangorde ?rangorde.
+  }
+}
+INSERT {
+  GRAPH ?graph {
+    ?mandataris mandaat:rangorde ?newLabel.
+  }
+}
+WHERE {
```
Comment (Rahien) [inline config/migrations/2024/20241108111400-change-rangorde-TS-to-schepen.sparql:15]: this query didn't work for me when i ran it on a prod backup

Comment (Rahien) [reply config/migrations/2024/20241108111400-change-rangorde-TS-to-schepen.sparql:15]: as in: there were cases of Toegevoegde schepen still left in the data

Comment (Rahien) [reply config/migrations/2024/20241108111400-change-rangorde-TS-to-schepen.sparql:15]: verify with:
```
PREFIX mandaat: <http://data.vlaanderen.be/ns/mandaat#>
SELECT * WHERE {
  GRAPH ?g {
  ?s mandaat:rangorde ?rank.
  FILTER(CONTAINS(?rank, "Toegevoegde schepen"))
  }
} 
```

File: config/migrations/2024/20241108111400-change-rangorde-GR-to-lid.sparql [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+  GRAPH ?graph {
+  ?mandataris mandaat:rangorde ?rangorde.
+  }
+}
+INSERT {
+  GRAPH ?graph {
+    ?mandataris mandaat:rangorde ?newLabel.
+  }
+}
+WHERE {
```
Comment (Rahien) [inline config/migrations/2024/20241108111400-change-rangorde-GR-to-lid.sparql:14]: this query didn't work for me when running on a prod backup, there are still cases of "Gemeenteraadslid" in the rangorde data. Verify with:
```
PREFIX mandaat: <http://data.vlaanderen.be/ns/mandaat#>
SELECT * WHERE {
  GRAPH ?g {
  ?s mandaat:rangorde ?rank.
  FILTER(CONTAINS(?rank, "Gemeenteraadslid"))
  }
} 
```

**Review by Rahien — CHANGES_REQUESTED** [review]

File: config/migrations/2024/20241108111400-change-rangorde-GR-to-lid.sparql (hunk 4 above)
Comment (JonasVanHoof) [reply config/migrations/2024/20241108111400-change-rangorde-GR-to-lid.sparql:14]: The mandatarissen that you get returned here for "Gemeenteraadslid" do not have Gemeenteraad classificatie.. The migration works for the classificatie for Gemeenteraad

These are the bestuursfunctie codes:
- https://data.vlaanderen.be/doc/concept/BestuursfunctieCode/5ab0e9b8a3b2ca7c5e000012 **Voorzitter van de gemeenteraad**
- https://data.vlaanderen.be/id/concept/BestuursfunctieCode/5ab0e9b8a3b2ca7c5e000015 **Lid van de Raad voor Maatschappelijk Welzijn**

File: config/migrations/2024/20241108111400-change-rangorde-TS-to-schepen.sparql (hunk 1 above)
Comment (JonasVanHoof) [reply config/migrations/2024/20241108111400-change-rangorde-TS-to-schepen.sparql]: Added

File: config/migrations/2024/20241108111400-change-rangorde-TS-to-schepen.sparql (hunk 3 above)
Comment (JonasVanHoof) [reply config/migrations/2024/20241108111400-change-rangorde-TS-to-schepen.sparql:15]: The result of your query is now with the update
```
[
	{
		"rank": "Toegevoegde schepen "
	},
	{
		"rank": "Toegevoegde schepen (derde)"
	}
]
```

The first one should be fixed by the rangorde input and the second one is something a user should fix IMO

Comment (JonasVanHoof) [reply config/migrations/2024/20241108111400-change-rangorde-TS-to-schepen.sparql:15]: Both queries now return no results so every match is handled, if the latest to load data is somewhat the same as on PROD it should handle it. If not i will just have to add ALL bestuursfunctie codes to the Values statement..

File: config/migrations/2024/20241108111400-change-rangorde-GR-to-lid.sparql (hunk 4 above)
Comment (JonasVanHoof) [reply config/migrations/2024/20241108111400-change-rangorde-GR-to-lid.sparql:15]: As above

**Review by JonasVanHoof — COMMENTED** [review]
SU coments

File: config/migrations/2024/20241108111400-change-rangorde-TS-to-schepen.sparql (hunk 3 above)
Comment (JonasVanHoof) [reply config/migrations/2024/20241108111400-change-rangorde-TS-to-schepen.sparql:15]: Will create a ticket for this

File: config/migrations/2024/20241108111400-change-rangorde-GR-to-lid.sparql (hunk 4 above)
Comment (JonasVanHoof) [reply config/migrations/2024/20241108111400-change-rangorde-GR-to-lid.sparql:15]: You can check if all items are replaced by just running the query with a select

```
SELECT ?rangorde ?newLabel
```

**Review by Rahien — APPROVED** [review]
no remaining cases in my prod backup

**Pull request merged**